canvas_modules/harness/cypress/e2e/canvas/node-insert-in-link.cy.js

```diff
@@ -200,6 +201,8 @@ describe("Test dropping a node, with existing links, on the canvas onto a link w
 	it("Drag node with output link from stream on the canvas onto a link - should 'work'", function() {
 		// Move the Sample node from the canvas onto the link between the
 		// Execution node and the Supernode.
+		// On the first Sort node moves and on the second move Sample node will move
+		cy.moveNodeToPosition("Sample", 400, 200);
```

Comment:
This look like another instance where the initial selection on the node doesn't work.  Please try replacing this 'moveNodeToPosition with the code we've used else where where we get the node and click on it.